gapis/gfxapi/resource.go

```diff
@@ -25,8 +25,11 @@ type ResourceMap map[Resource]id.ID
 
 // Resource represents an asset in a capture.
 type Resource interface {
-	// ResourceName returns the UI name for the resource.
-	ResourceName() string
+	// ResourceName returns the UI identity for the resource.
```

Comment:
I'm really not sure about these changes. ResourceName was supposed to be the UI label. If you want a unique key for the resource, that's the path to the resource.
`ResourceName()` returns values that are not unique between resource types, nor resources of the same type. I don't see what purpose it serves.